packages/kit/src/runtime/client/client.js

```diff
@@ -11,14 +11,22 @@ import {
 	initial_fetch,
 	notifiable_store,
 	scroll_state
-} from './utils';
+} from './utils.js';
+import { parse } from './parse.js';
 
 import Root from '__GENERATED__/root.svelte';
-import { routes, fallback } from '__GENERATED__/manifest.js';
+import { components, dictionary, validators } from '__GENERATED__/client-manifest.js';
 
 const SCROLL_KEY = 'sveltekit:scroll';
 const INDEX_KEY = 'sveltekit:index';
 
+const routes = parse(components, dictionary, validators);
+
+// we import the root layout/error components eagerly, so that
+// connectivity errors after initialisation don't nuke the app
+const default_layout = components[0]();
```

Comment:
is this part of the validators effort or something different? 
what happens when one of them errors?

Reply:
it's just moving this existing logic https://github.com/sveltejs/kit/blob/fac284475decb1531ee9c2413f58020256f719d2/packages/kit/src/core/sync/write_manifest.js#L69-L71